conf/application.conf

```diff
@@ -161,4 +161,7 @@ is-shuttered = false
 timeout-dialog {
   timeout = 900
   countdown = 120
-}
\ No newline at end of file
+}
+
+# New Accessibility Statement platform path
+accessibility-statement.service-path = "/vat-repayment-tracker"
```

Comment:
Is it a good config?

In my opinion it should be `/vat-repayment-tracker/vat-repayment-accessibility-statement`

See how the config is used:

```  val accessibilityStatementHost: Option[String] =
    platformHost.orElse(config.getOptional[String]("accessibility-statement.host"))
  val accessibilityStatementPath: Option[String] =
    config.getOptional[String]("accessibility-statement.path")
  val accessibilityStatementServicePath: Option[String] =
    config.getOptional[String]("accessibility-statement.service-path")

  def url(implicit request: RequestHeader): Option[String] =
    for {
      host        <- accessibilityStatementHost
      path        <- accessibilityStatementPath
      servicePath <- accessibilityStatementServicePath
    } yield {
      s"$host$path$servicePath$query"
    }
```